src/__tests__/comment.test.ts

```diff
@@ -15,31 +16,57 @@ describe(`comment`, () => {
     with: text(),
   });
 
-  const db = defineDb({ foo, bar }, () => Promise.resolve({ rows: [], affectedCount: 0 }));
+  let lastQuery: string;
```

Comment:
Minor: Could combine these, e.g.

```ts
let lastExecuteFnArgs: {text: string, parameters: Array<string>} | null = null;
```

And then might be good for `checkQueryAsync` to clear them, just to protect against bugs where someone forgets to call it, e.g.

```ts
expect(lastExecuteFnArgs).toStrictEqual(expected);
lastExecteFnArgs = null;
```